src/components/actions/getApplicationDataRoute.ts

```diff
@@ -0,0 +1,11 @@
+import { getApplicationData } from './getApplicationData'
+import { combineRequestParams } from '../utilityFunctions'
+
+export const routeGetApplicationData = async (request: any, reply: any) => {
+  const { applicationId, reviewId } = combineRequestParams(request, 'camel')
+  const appDataParams: { applicationId: number; reviewId?: number } = {
+    applicationId: Number(applicationId),
+  }
+  if (reviewId) appDataParams.reviewId = Number(reviewId)
+  reply.send(await getApplicationData(appDataParams))
+}
```

Comment:
I just separated the routes used by the app proper (i.e. `routeGetApplicationData`) from those that are dev tools only (`runAction`, this one).

I really don't want someone to accidentally start using one of the dev routes in the actual app.